### DISCUSSION TITLE
Gui design

### DISCUSSION THREAD

**Discussion by vicky604** [tcunit/TcUnit#130] [Q&A] [answered]
Who can help me with this.🥺

You are required to Design following GUI output by Java code. Also, when a user type text in the text field(at left side of send button) and presses send button then the typed text must be vanished from the text field and appears in the text area(above button and text field). Furthermore, each newly entered text must be appeared on newlines of text area and a Scrollbar must be appeared on the text area when it is filled so that we can see old sent texts.

**Comment by sagatowski** [comment, marked as answer]
@vicky604 I think this is the wrong discussion-forum for your type of question. I'd recommend using StackOverflow instead.
https://stackoverflow.com/questions/5118701/how-to-create-a-gui-in-java
